client/client_impl.go

```diff
@@ -28,28 +28,30 @@ const (
 	pathLogs     = "%s/api/queue/logs/%d"
 	pathLogsAuth = "%s/api/queue/logs/%d?access_token=%s"
 
-	pathSelf        = "%s/api/user"
-	pathFeed        = "%s/api/user/feed"
-	pathRepos       = "%s/api/user/repos"
-	pathRepo        = "%s/api/repos/%s/%s"
-	pathChown       = "%s/api/repos/%s/%s/chown"
-	pathEncrypt     = "%s/api/repos/%s/%s/encrypt"
-	pathBuilds      = "%s/api/repos/%s/%s/builds"
-	pathBuild       = "%s/api/repos/%s/%s/builds/%v"
-	pathJob         = "%s/api/repos/%s/%s/builds/%d/%d"
-	pathLog         = "%s/api/repos/%s/%s/logs/%d/%d"
-	pathKey         = "%s/api/repos/%s/%s/key"
-	pathSign        = "%s/api/repos/%s/%s/sign"
-	pathRepoSecrets = "%s/api/repos/%s/%s/secrets"
-	pathRepoSecret  = "%s/api/repos/%s/%s/secrets/%s"
-	pathTeamSecrets = "%s/api/teams/%s/secrets"
-	pathTeamSecret  = "%s/api/teams/%s/secrets/%s"
-	pathNodes       = "%s/api/nodes"
-	pathNode        = "%s/api/nodes/%d"
-	pathUsers       = "%s/api/users"
-	pathUser        = "%s/api/users/%s"
-	pathBuildQueue  = "%s/api/builds"
-	pathAgent       = "%s/api/agents"
+	pathSelf          = "%s/api/user"
+	pathFeed          = "%s/api/user/feed"
+	pathRepos         = "%s/api/user/repos"
+	pathRepo          = "%s/api/repos/%s/%s"
+	pathChown         = "%s/api/repos/%s/%s/chown"
+	pathEncrypt       = "%s/api/repos/%s/%s/encrypt"
+	pathBuilds        = "%s/api/repos/%s/%s/builds"
+	pathBuild         = "%s/api/repos/%s/%s/builds/%v"
+	pathJob           = "%s/api/repos/%s/%s/builds/%d/%d"
+	pathLog           = "%s/api/repos/%s/%s/logs/%d/%d"
+	pathKey           = "%s/api/repos/%s/%s/key"
+	pathSign          = "%s/api/repos/%s/%s/sign"
+	pathRepoSecrets   = "%s/api/repos/%s/%s/secrets"
+	pathRepoSecret    = "%s/api/repos/%s/%s/secrets/%s"
+	pathTeamSecrets   = "%s/api/teams/%s/secrets"
+	pathTeamSecret    = "%s/api/teams/%s/secrets/%s"
+	pathGlobalSecrets = "%s/api/global/secrets"
+	pathGlobalSecret  = "%s/api/global/secret/%s"
```

Comment:
Isn't that supposed to be `"%s/api/global/secrets/%s"` instead of `"%s/api/global/secret/%s"` ? just to keep it consistent.